src/Microsoft.VisualStudio.AppDesigner/PropPages/PropertyPageException.vb

```diff
@@ -33,9 +33,9 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             Me.HelpLink = helpLink
         End Sub
 
-        Public Sub New(message As String, innerException As Exception, ShowHeaderandFooterInErrorControl As Boolean)
+        Public Sub New(message As String, innerException As Exception, ShowHeaderAndFooterInErrorControl As Boolean)
```

Comment:
This is *technically* a source breaking change, but I'm willing to live with it if anyone is consuming this and using named parameters ;)

Reply:
Also can be a runtime one too if you are using dynamic binding, but I also came to the same conclusion. Make note that editors internals are public to the flavors as well, so technically also breaking when we change internals. I've ignored it for the most part as they really only use the property page infrastructure.

Reply:
What's the mechanism that allows internal members to be public to the flavors?

Reply:
InternalsVisible listed under `My Project\AssemblyInfo.vb`.

Reply:
I see. We own them and they're a closed set, right? We're not talking about breaking customers here are we?

Reply:
No, we own them. Don't worry about them - just making you aware that they exist and they tend to use Property Page functionality.